### PR TITLE
update-readme-with-new-dashboards-added

### PR DESCRIPTION
**Pull request by ericanagle1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,20 +6,34 @@ imported into JupiterOne Insights app.
...
+- [Jira Insights](src/boards/jira-insights/)
+- [Network Security](src/boards/network-security/)
+- [Okta User Management](src/boards/okta-user-management/)
 - [Resource Classification](src/boards/resource-classification/)
 - [Risk Register](src/boards/risk-register/)
 - [Software Package Dependencies and Licenses](src/boards/code-deps-licenses/)
 - [Team Growth](src/boards/team-growth/)
+- [Team Management and Direct Repors](src/boards/team-manager-direct-reports/)
 - [User Access](src/boards/user-access/)
+- [User Endpoint Blast Radius](src/boards/user-endpoint-blast-radius/)
```
Comment (erkangz) [inline README.md:36]: let's prefix this with "IR"

File: README.md [hunk 2]
```diff
@@ -6,20 +6,34 @@ imported into JupiterOne Insights app.
 List of dashboards:
 
 - [AWS Accounts](src/boards/aws-accounts/)
+- [AWS Cost Analysis](src/boards/aws-cost-analysis)
 - [AWS IAM](src/boards/aws-iam/)
+- [AWS Resources](src/boards/aws-resources/)
 - [AWS S3 Security](src/boards/aws-s3-security/)
+- [Azure DataStore Resources](src/boards/azure-datastore-security/)
+- [Azure Resources](src/boards/azure-resources/)
+- [Cloud Instance & Workload Analysis](src/boards/cloud-instance-workload-analysis)
```
Comment (erkangz) [inline README.md:15]: let's prefix this with "IR" and move next to the other IR dashboard because they are of the same use case.

**Review by erkangz — CHANGES_REQUESTED** [review]

**Review by ericanagle1 — COMMENTED** [review]
Updated titles for IR dashboards

File: README.md [hunk 3]
```diff
@@ -6,19 +6,33 @@ imported into JupiterOne Insights app.
...
+- [IR - User Endpoint Blast Radius](src/boards/user-endpoint-blast-radius/)
+- [Jamf & CrowdStrike Sensor Activity](src/boards/jamf-crowdstrike/)
+- [Jira Insights](src/boards/jira-insights/)
+- [Network Security](src/boards/network-security/)
+- [Okta User Management](src/boards/okta-user-management/)
 - [Resource Classification](src/boards/resource-classification/)
 - [Risk Register](src/boards/risk-register/)
 - [Software Package Dependencies and Licenses](src/boards/code-deps-licenses/)
 - [Team Growth](src/boards/team-growth/)
+- [Team Management and Direct Repors](src/boards/team-manager-direct-reports/)
```
Comment (erkangz) [inline README.md:35]:
```suggestion
- [Team, Manager and Direct Reports](src/boards/team-manager-direct-reports/)
```

**Review by ericanagle1 — COMMENTED** [review]
Let me know if this looks a little better

**Review by DrewJohnsonGT — APPROVED** [review]

**Review by erkangz — APPROVED** [review]

**Pull request merged**